src/hooks/useMaxVisibleSections.ts

```diff
@@ -1,4 +1,4 @@
-import { useEffect, useState } from 'react';
+import React, { useEffect, useState } from 'react';
```

Comment:
why `React` import is needed?

Reply:
React import should be there on every react component

Reply:
The hook does not require React import, only needed if you use any JSX.

Reply:
line no 4,5 have React.MutableRefObject, so it needs the react import